src/c++/uds/kernelLinux/tests/getTestIndexNames.c

```diff
@@ -74,20 +87,25 @@ struct block_device **getTestMultiBlockDevices(void)
 /**********************************************************************/
 void putTestBlockDevice(struct block_device *bdev)
 {
-#ifndef VDO_USE_ALTERNATE
+
```

Comment:
Please remove this line. We do not need a blank line at the beginning of a function.